docs/reference/commandline/build.md

```diff
@@ -345,6 +345,13 @@ This will use a file called `Dockerfile.debug` for the build instructions
 instead of `Dockerfile`.
 
 ```bash
+$ docker build -f - . < ../Dockerfile
```

Comment:
should the example use something like `echo` / `cat` to be clearer it's from `stdin`? The current example looks very similar to

```
docker build -f ../Dockerfile .
```